app/config/variables.php

```diff
@@ -718,6 +718,15 @@
                 'question' => '',
                 'filter' => ''
             ],
+            [
+                'name' => '_APP_FUNCTIONS_SIZE_LIMIT',
```

Comment:
we already have the same variable above?

Reply:
whoops, fixed